src/app/screens/Home/DefaultView/index.tsx

```diff
@@ -161,7 +156,14 @@ const DefaultView: FC<Props> = (props) => {
 
     setIncomingTransactions(invoices);
     setIsLoadingInvoices(false);
-  };
+  }, [getFormattedFiat, settings]);
```

Comment:
`settings.showFiat` would suffice right?

Reply:
Good suggestion settings.showFiat is better